File: scripts/base64.rb
#!/usr/bin/ruby

#
# Use: ./base64.rb
# Heitor Gouvêa - hi@heitorgouvea.me

require 'base64'

def main
  if ARGV[0]
    $decryptBase64 = Base64.decode64(ARGV[0])

    puts $decryptBase64
  end
end

main()
exit

File: crypto/base64.pl
#!/usr/bin/perl

use 5.010;
use strict;
use warnings;
use MIME::Base64;

sub main {
  my $string = $ARGV[0];

  if ($string) {
    my $decoded = MIME::Base64::decode($string);
    
    print $decoded;
  }
}

main();
exit;
